fladgejt/webui/predmety.py

```diff
@@ -17,12 +18,11 @@ def _open_register_predmetov(self):
     def get_informacny_list(self, kod_predmetu, akademicky_rok):
         app = self._open_register_predmetov()
 
-        index = find_option(
-            app.d.akRokComboBox.options, title=akademicky_rok)
-        app.d.akRokComboBox.select(index)
+        stredisko, skratka, _ = decode_key(kod_predmetu)[0].split('/')
```

Comment:
Á, super! Dá sa filtrovať podľa strediska...
Pozor - myslím, že skratka môže obsahovať '/'. Preto tam pôvodne bol ten `'/'.join(...)`.
Možno radšej niečo takéto?
```py
stredisko, _, zvysok = decode_key(kod_predmetu)[0].partition('/')
skratka = zvysok.rpartition('/')[0]
```
Mimochodom: V skutočnosti sú aj predmety, ktoré sa líšia iba v tom poslednom čísle. :-( Tuším je to niečo ako rok vydania tejto verzie. Skúsim sa zamyslieť, čo s nimi, ale asi to môže zostať ako issue na neskôr.